rootbeer/constant-source/test-constant-source.html

```diff
@@ -0,0 +1,183 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Rootbeer Test: ConstantSource</title>
+    <script src="../resources/testharness.js"></script>
+    <script src="../resources/testharnessreport.js"></script>
+    <script src="../resources/audit.js"></script>
```

Comment:
I am not sure about this. Do we want these things in this project? Do you have a plan to integrate this project to Chromium at some point?

Reply:
As I mentioned, it's not necessary, but it makes it easier to pull in WPT tests to test this node.  Perhaps that's not important enough to have these, but writing tests take a long time and building off of existing tests would be useful.

Reply:
That's a good point.

But then how do we pull these libraries when they are changed? (version update?) Also I don't see any script that handles dependencies. Manually copying files around is something that we don't want to do...

Also running a test seems very manual in the current form. Do you have a plan to run multiple tests with a script in the future?